test/system/035-logs.bats

```diff
@@ -434,4 +434,26 @@ EOF
     run_podman rm $cname
 }
 
+@test "podman run -l passthrough-tty" {
+    skip_if_remote
+
+    if ! tty -s; then
+        skip "not running in a tty"
+    fi
```

Comment:
looking at CI this seems to always trigger so it is unable to catch any regression in CI thus effectively making the test pointless. Can we setup a tty for this test somehow?